src/main/java/nl/tudelft/contextproject/model/Drawable.java

```diff
@@ -8,7 +8,7 @@
  * Interface representing a Drawable object.
  * This object has a geometry that can be updated.
  */
-public interface Drawable {	
+public interface Drawable extends Collidable {	
```

Comment:
@chiel prefered it this way. A collidable does not mean that the entire object is solid. I purpose a name change of the interface to physicalinteraction